src/hotspot/os/windows/os_windows.cpp

```diff
@@ -860,6 +860,8 @@ bool os::has_allocatable_memory_limit(size_t* limit) {
 }
 
 int os::active_processor_count() {
+  win32::log_general_system_processor_debug_info();
```

Comment:
Please move this to the init function as we don't want to have to check logging on each call. We don't need a separate function in that case. Thanks